modules/gltf/test/gltf-writer.spec.js

```diff
@@ -52,3 +80,37 @@ test('GLTFWriter#encode', async t => {
 
   t.end();
 });
+
+test('GLTFWriter#encode buffers', async t => {
```

Comment:
Better name? (test scenegraph builder functions)?

Reply:
Done in https://github.com/visgl/loaders.gl/pull/970